samples/MultiMasterOperations/MultiMasterOperations.pyproj

```diff
@@ -11,7 +11,7 @@
     <OutputPath>.</OutputPath>
     <Name>MultiMasterOperations</Name>
     <RootNamespace>MultiMasterOperations</RootNamespace>
-    <InterpreterId>Global|PythonCore|3.4</InterpreterId>
+    <InterpreterId>Global|PythonCore|3.6</InterpreterId>
```

Comment:
pleas retain the interpreter version as 3.4 as that it the minimum version supported by the sdk